.github/workflows/develop-status.yml

```diff
@@ -19,6 +19,13 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: ${{ matrix.jdk }}
+    - name: Cache local Maven repository
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
```

Comment:
What is the difference between key and restore key? Are they supposed to be different?

Reply:
See https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows . I just copy-pasted this setup. As I inderstand it the restore keys are optional, and can be used as alternative lookup keys if the main key gives no hit.